src/core/ext/filters/client_channel/subchannel.cc

```diff
@@ -77,6 +79,7 @@ typedef struct external_state_watcher {
 } external_state_watcher;
 
 struct grpc_subchannel {
+  intptr_t uuid;
```

Comment:
i remember discussing if we need to save uuid here. don't remember if there was a specific reason why we wanted it

Reply:
Done